### PR TITLE
Css structure refactor

### LINKED ISSUE #46: Refatorar arquitetura de estilos
Hoje temos 3 níveis de variação que um cartão pode ter: 
- banco
- modelo
- tipo

e a lógica que utilizamos foi ter uma classe banco e outra classe concatenando os 3: banco-modelo-tipo. Mas aqui estamos repetindo o banco e evitando coisas compartilhadas

Exemplo: todos os cartões bancoX podem ter um logo só;
a maioria dos cartões modeloY tem um tipo de bandeira, mas pode ser diferente para outros casos.

A solução atual permite que sigamos a regra do styleLint de que não podemos ter mais de 2 condições (não permitindo `.banco .modelo .tipo`.

Há duas coisas possíveis a serem feitas:
- Remover regra do lint e refatorar para que usemos o padrão definido acima, já que o lint nesse caso não se aplica.
- css-modules

### PR DESCRIPTION
**Pull request by vcapretz**
## Description

We are now using css-modules to avoid conflicts with classes and allow new stuff such as composing classes 

closes #46 

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by GitCop** [comment]
Os seguintes commits de seu Pull Request precisam de ajustes no título:

 * Commit: e711b44edd5901afaf7ccdafe49661c5396f9ed4
  * Utilize o padrão `escopo: ação descrição`, sempre em minúsculo e inglês.  
  
 - O **escopo** não deve conter espaços. Se a palavra for composta, separe por traços  
    - A **ação** deve ser **add**, **remove**, **rename**, **refactor**, **move**,  **bump**, **update** ou **release**, **fix**  
    - A **descrição** deve conter uma rápida descrição da alteração

O guia completo de estilo de commits está disponível em https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: src/components/PaymentCard/index.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ PaymentCard.defaultProps = {
   bank: '',
   model: '',
   type: '',
-  brand: '',
+  brand: 'mastercard',
```
Comment (thiagommedeiros) [inline src/components/PaymentCard/index.js]: why does the default brand need to be `mastercard`?

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "eslint-plugin-react": "7.4.0",
     "file-loader": "1.1.5",
     "gh-pages": "1.0.0",
+    "identity-obj-proxy": "^3.0.0",
```
Comment (augusto-jm-amaral) [inline package.json]: you need to remove this caret.

**Comment by augusto-jm-amaral** [comment]
With css modules it looks like everything has cleared, Nice @vcapretz!

File: src/components/PaymentCard/index.js (hunk 1 above)
Comment (vcapretz) [reply src/components/PaymentCard/index.js]: it doesn't! fixed

File: package.json (hunk 2 above)
Comment (vcapretz) [reply package.json]: done, thanks!

**Pull request merged**